src/shims/foreign_items.rs

```diff
@@ -102,6 +103,39 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
         }
     }
 
+    /// Lookup the body of a non-external foreign item.
+    fn lookup_non_external_foreign_item(
```

Comment:
"non-external foreign" is a strange term and not standard terminology, either.

What about `lookup_exported_symbol`?

Reply:
Is there a standard term? I would like to rename tests and change the PR title.

Reply:
I don't know, but something involving "exported" makes sense IMO.

Reply:
"exported symbol calls"?

Reply:
This is hard to parse ("calls what"? 'calls' might be a verb. ;)

What about "Implement calls to exported symbols"?

Reply:
> What about "Implement calls to exported symbols"?

But... how to embed that five words in file names for tests?

Reply:
Hm... `compile-fail/functions_calls/exported_symbol_*`?